lib/determine_outcome.rb

```diff
@@ -1,27 +1,29 @@
 # frozen_string_literal: true
 
-class CheckGameState
+class DetermineOutcome
   def initialize(game_state_gateway:)
     @game_state_gateway = game_state_gateway
     @winning_sets = [
-      [0,1,2],
-      [3,4,5],
-      [6,7,8],
-      [0,3,6],
-      [1,4,7],
-      [2,5,8],
-      [0,4,8],
-      [6,4,2]
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 4, 8],
+      [6, 4, 2]
     ]
   end
 
   def execute
     board = @game_state_gateway.retrieve
-    check_for_winner(board)
+    winner = check_for_winner(board)
+    winner = 3 if check_for_draw(board) == 9
+    return winner
```

Comment:
Unnecessary `return`. Line 22 can just be `winner`.

Reply:
removed it, thanks 😄